studio/app/optinist/wrappers/caiman/cnmf.py

```diff
@@ -116,7 +118,7 @@ def caiman_cnmf(
         backend="local", n_processes=None, single_thread=True
     )
 
-    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=None, params=ops)
+    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=Ain, params=ops)
```

Comment:
@quanpython 
Please obtain the following evidence. (Output of W, b0)

- https://caiman.readthedocs.io/en/master/Getting_Started.html#variables-for-1p-processing-cnmf-e
  > The variables for one photon processing are the same, with an additional variable estimates.W for the matrix that is used to compute the background W for the matrix that is used to compute the background, and estimates.b0 for the baseline value for each pixel.

Reply:
@itutu-tienday
How do you want to check them? (Print it out?)
Currently, when I run cnmfe, value of W and  b0 is None. Does their value depend on the parameters?

Reply:
@itutu-tienday
I updated some params name in `caiman_cnmfe.yaml` and  `W`, `b0` value  were returned.
But the `estimate.b` value was `None`, I adjusted some code to handle it.

Reply:
@quanpython 
> How do you want to check them? (Print it out?)

We assume the final generated nwb file, or a screenshot of the file contents, etc. (Check the output of W,b0)

> Currently, when I run cnmfe, value of W and b0 is None. Does their value depend on the parameters?

> I updated some params name in caiman_cnmfe.yaml and W, b0 value were returned.

Thanks for the survey.
At this point, we do not know which combination of parameters is responsible for the output, but what was the output result from the following sample program?
- https://github.com/flatironinstitute/CaImAn/blob/main/demos/notebooks/demo_pipeline_cnmfE.ipynb

Reply:
@itutu-tienday
I added W,b0 into nwbfile, you can check it in `/processing/ophys/estimates.W` and  `/processing/ophys/estimates.b0`.